src/backend/backend.py

```diff
@@ -39,14 +41,16 @@ def __init__(self):
 
     @staticmethod
     def _init_websocket():
-        while True:
+        app_started = False
+        while not app_started:
             try:
                 app.run()
-                break
+                app_started = app._started
             except AlreadyRunningError:
                 app.stop()
-            except Exception:
```

Comment:
I understand why we had it this way, but I still think we should force it to be more explicit.
Because with a catch as broad as `Exception` you would also have things like `SyntaxError` getting caught silently.
So I'd rather make it explicit as it documents possible failures. And if we miss one we can add it later